vm/devices/vmbus/vmbus_proxy/src/proxyioctl.rs

```diff
@@ -138,6 +138,7 @@ pub struct VMBUS_PROXY_RESTORE_CHANNEL_INPUT {
     pub InterfaceType: GUID,
     pub InterfaceInstance: GUID,
     pub SubchannelIndex: u16,
+    pub TargetVtl: u8,
```

Comment:
If we're going to be stuffing extra stuff in the padding in the future, it would be good to add in explicit padding fields here for everything now so that we make sure we're passing zeros.